src/sdk.ts

```diff
@@ -455,13 +437,13 @@ export class OpenSeaSDK {
   }: {
     asset: Asset;
     accountAddress: string;
-    startAmount: BigNumberInput;
-    endAmount?: BigNumberInput;
-    quantity?: BigNumberInput;
+    startAmount: BigNumberish;
+    endAmount?: BigNumberish;
```

Comment:
These amounts are gwei (whole numbers) not eth right?

Reply:
right, wei not gwei